src/core/load-fragment.c

```diff
@@ -4575,6 +4575,11 @@ int config_parse_ip_filter_bpf_progs(
         assert(rvalue);
         assert(paths);
 
+#if !(HAVE_LIBBPF)
+        log_syntax(unit, LOG_ERR, filename, line, -1, "libbpf is not found, ignoring %m");
```

Comment:
do not make up error codes... i.e. no -1 please. use 0 here...
also why have this ifdeffery at all here? when bpf is disabled we should still compile the parsers for this, and this function at its end already covers the non-bpf case...